transfer.md

```diff
@@ -41,7 +41,8 @@ There are other equivalents for these tools
 | :---: | :---: |
 | `docker attach` | [`kpod exec`](./docs/kpod-attach.1.md) ***|
 | `docker build`  | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
-| `docker cp`     | [`kpod mount`](./docs/kpod-cp.1.md) ****   |
+| `docker cp`     | [`kpod mount`](./docs/kpod-cp.1.md) ***   |
```

Comment:
This should have 4 asterisks  "****".

Also the note at the bottom of the table is having a carriage return issue.  If you could try touching that up that would be great.

Reply:
fixed the ****, didnt see the CR issue.